src/components/common/InputForm.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {View, Text, StyleSheet, TextInput } from 'react-native';
+
+const InputForm = ({title, secureTextEntry}) => (
+    <View style = { styles.container }> 
```

Comment:
```suggestion
    <View style ={styles.container}> 
```
Apply this suggestion to all the passing props that you made